doc/man/dune-build.1

```diff
@@ -0,0 +1,346 @@
+NAME
+       dune-build - Build the given targets, or all installable targets if
+       none are given.
+
+SYNOPSIS
+       dune build [OPTION]… [TARGET]…
+
+DESCRIPTION
+       Targets starting with a @ are interpreted as aliases.
+
+OPTIONS
+       --action-stderr-on-success=VAL
+           Same as --action-stdout-on-success but for the standard output for
+           error messages. A good default for large mono-repositories is
+           --action-stdout-on-success=swallow
+           --action-stderr-on-success=must-be-empty. This ensures that a
+           successful build has a "clean" empty output.
+
+       --action-stdout-on-success=VAL
+           Specify how to deal with the standard output of actions when they
+           succeed. Possible values are: print to just print it to Dune's
+           output, swallow to completely ignore it and must-be-empty to
+           enforce that the action printed nothing. With must-be-empty, Dune
+           will consider that the action failed if it printed something to
+           its standard output. The default is print.
+
+       --build-info
+           Show build information.
+
+       --error-reporting=VAL (absent=deterministic)
+           Controls when the build errors are reported. early - report errors
+           as soon as they are discovered. deterministic - report errors at
+           the end of the build in a deterministic order. twice - report each
+           error twice: once as soon as the error is discovered and then
+           again at the end of the build, in a deterministic order.
+
+       -f, --force
+           Force actions associated to aliases to be re-executed even if
+           their dependencies haven't changed.
+
+       --file-watcher=VAL (absent=automatic)
+           Mechanism to detect changes in the source. Automatic to make dune
+           run an external program to detect changes. Manual to notify dune
+           that files have changed manually."
+
+       --passive-watch-mode
+           Similar to [--watch], but only start a build when instructed
+           externally by an RPC.
+
+       --react-to-insignificant-changes
+           react to insignificant file system changes; this is only useful
+           for benchmarking dune
+
+       --sandbox=VAL (absent DUNE_SANDBOX env)
+           Sandboxing mode to use by default. Some actions require a certain
+           sandboxing mode, so they will ignore this setting. The allowed
+           values are: none, symlink, copy, hardlink.
+
+       -w, --watch
+           Instead of terminating build after completion, wait continuously
+           for file changes.
+
+       --wait-for-filesystem-clock
```

Comment:
I don’t understand what this is saying. Does `—wait-for-filesystem-clock` create a Dune digest file?